src/electionguard_cli/cli_steps/output_step_base.py

```diff
@@ -0,0 +1,47 @@
+from typing import List, Any
+from os.path import join
+
+from electionguard import to_file
+from electionguard.guardian import Guardian, GuardianRecord
+from electionguard_tools.helpers.export import GUARDIAN_PREFIX
+
+from .cli_step_base import CliStepBase
+from ..cli_models import CliElectionInputsBase
+
+
+class OutputStepBase(CliStepBase):
+    """Responsible for common functionality across all CLI commands related to outputting results."""
+
+    def _export_private_keys(self, output_keys: str, guardians: List[Guardian]) -> None:
+        if output_keys is None:
+            return
+
+        private_guardian_records = [
+            guardian.export_private_data() for guardian in guardians
+        ]
+        file_path = output_keys
+        for private_guardian_record in private_guardian_records:
+            file_name = GUARDIAN_PREFIX + private_guardian_record.guardian_id
+            to_file(private_guardian_record, file_name, file_path)
+        self.print_value("Guardian private keys", output_keys)
+        self.print_warning(
+            f"The files in {file_path} are secret and should be protected securely and not shared publicly."
+        )
+
+    @staticmethod
+    def _get_guardian_records(
+        election_inputs: CliElectionInputsBase,
+    ) -> List[GuardianRecord]:
+        return [guardian.publish() for guardian in election_inputs.guardians]
+
+    def _export_file(
```

Comment:
This should be a public method and definitely placed in its own module. Perhaps an export module that just does this. Maybe even in `electionguard_tools`

Reply:
But it's specific to the CLI because of the `self.print_value` call.  Do you want me to just move lines 45 and 46 into `electionguard_tools`?

Reply:
The file stuff and location has slowly been creeping in to all these setups. I'm not a huge fan. The to_file already does the location concatenation within that function. That could just return the location. Then that line wouldn't be necessary.